packages/analytics-js-integrations/src/integrations/Amplitude/browser.js

```diff
@@ -85,14 +85,23 @@ class Amplitude {
 
     // rudderElement.message.context will always be present as part of identify event payload.
     const { traits } = rudderElement.message.context;
-    const { userId } = rudderElement.message;
-
+    const { userId, integrations } = rudderElement.message;
+    const amplitudeIntgConfig = getDestinationOptions(integrations);
+    const fieldsToUnset = amplitudeIntgConfig?.fieldsToUnset || undefined;
+    const amplitudeIdentify = new window.amplitude.Identify();
+    let sendIdentifyCall = false;
+    if (fieldsToUnset) {
```

Comment:
is it possible that it can be empty array?

Reply:
Yes it is possible

Reply:
Should this be changed to `Array.isArray(fieldsToUnset)` in that case ?

Reply:
should we also check for length > 0?

Reply:
Done

Reply:
Don't think so many conditions are required. Just `Array.isArray()` is actually sufficient. But it's fine

Reply:
What about length?

Reply:
If type is array
```javascript
> a = []
[]
> a.forEach(console.log)
undefined
```
Nothing would happen

Reply:
what about `sendIdentifyCall = true;`

Reply:
Acha.. Got it 👍🏼